src/hooks/useLocalStorage.ts

```diff
@@ -60,6 +63,9 @@ export const useLocalStorage = <T>({
     (value) => {
       storageSet(key, value);
       setState(value);
+      if (expire) {
+        debounce(storageService.remove, expire * 60)(key);
+      } 
```

Comment:
I'm not sure if I understand this logic. If we want to make this value expire after some period of time, then we'd have to store the expiry in an additional timestamp item in local storage instead of using it like that. This would now only work if you keep the app open. 
Also, why multiply with `60`? The `expire` value is in seconds but the `debounce` value is in milliseconds so we would have to multiply with `1000`.

Reply:
You are totally right, this only works with the app working. I'm not sure how the 60 got there, probably I was testing and needed a lower value to see the effects, but weird.
I removed this functionality from this branch, but created another PR with an improved version here https://github.com/pendulum-chain/portal/pull/186

Reply:
BTW, we don't need the expire date anymore, so it wasn't needed anyways here.

Reply:
Got it, thanks :+1: